.github/workflows/test-install-script.yml

```diff
@@ -13,13 +13,17 @@ jobs:
     runs-on: ${{ matrix.runs_on }}
     strategy:
       matrix:
+        fail-fast: false
```

Comment:
I think `fail-fast` may need to be 1 level up ; [according to the documentation](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast) it's `strategy.fail-fast`.

Reply:
Good catch, thanks!

Reply:
Fixed!